### PR TITLE
[JENKINS-57803] Add readme

### PR DESCRIPTION
**Pull request by baymac**
Added a instructions related to Jenkins in general and plugin usage.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+* GitLab Server Plugin
+* GitLab Branch Source Plugin
+
+To fully be able to run a Jenkins Continuous Integration on a GitLab repository or project, you require three plugins:
+
+1. [GitLab API Plugin](https://github.com/jenkinsci/gitlab-api-plugin) - Wraps GitHub Java API
+
+2. [GitLab Plugin](https://github.com/jenkinsci/gitlab-plugin/) - Server configuration and web hooks Management 
+
+3. GitLab Branch Source Plugin - To support Multi branch Pipeline Jobs (including Merge Requests) and Folder organisation
```
Comment (markjacksonfishing) [inline README.md]: Capitalize branch

File: README.md [hunk 2]
```diff
@@ -0,0 +1,336 @@
...
+
+## Jenkins and GitLab Integration:
+
+This section contains information related to how GitLab and Jen
```
Comment (markjacksonfishing) [inline README.md]: `Jen`?

**Review by markjacksonfishing — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (baymac) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (baymac) [reply README.md]: fixed

**Pull request merged**